src/rendering/common.jl

```diff
@@ -0,0 +1,174 @@
+
+    # Only use floats if chunk has caption or sets fig_env
+    if !isnothing(chunk.options[:fig_cap]) && isnothing(chunk.options[:fig_env])
+        (chunk.options[:fig_env] = docformat.fig_env)
+    end
+
+    hasproperty(docformat, :indent) && (chunk.content = indent(chunk.content, docformat.indent))
+
+    chunk.content = format_code(chunk.content, docformat)
+
+    if !chunk.options[:eval]
+        return if chunk.options[:echo]
+            string(docformat.codestart, '\n', chunk.content, docformat.codeend)
+        else
+            ""
+        end
+    end
+
+    if chunk.options[:term]
+        result = format_termchunk(chunk, docformat)
+    else
+        result = if chunk.options[:echo]
+            # Convert to output format and highlight (html, tex...) if needed
+            string(docformat.codestart, chunk.content, docformat.codeend, '\n')
+        else
+            ""
+        end
+
+        if (strip(chunk.output) ≠ "" || strip(chunk.rich_output) ≠ "") &&
+           (chunk.options[:results] ≠ "hidden")
+            if chunk.options[:results] ≠ "markup" && chunk.options[:results] ≠ "hold"
+                strip(chunk.output) ≠ "" && (result *= "$(chunk.output)\n")
+                strip(chunk.rich_output) ≠ "" && (result *= "$(chunk.rich_output)\n")
+            else
+                if chunk.options[:wrap]
+                    chunk.output =
+                        '\n' * wraplines(chunk.output, chunk.options[:line_width])
+                    chunk.output = format_output(chunk.output, docformat)
+                else
+                    chunk.output = '\n' * rstrip(chunk.output)
+                    chunk.output = format_output(chunk.output, docformat)
+                end
+
+                if hasproperty(docformat, :indent)
+                    chunk.output = indent(chunk.output, docformat.indent)
+                end
+                strip(chunk.output) ≠ "" && (
+                    result *= "$(docformat.outputstart)$(chunk.output)\n$(docformat.outputend)\n"
+                )
+                strip(chunk.rich_output) ≠ "" && (result *= chunk.rich_output * '\n')
+            end
+        end
+    end
+
+    # Handle figures
+    if chunk.options[:fig] && length(chunk.figures) > 0
+        if chunk.options[:include]
+            result *= formatfigures(chunk, docformat)
+        end
+    end
+
+    return result
+end
+
+format_code(code, docformat) = code
+
+format_output(result, docformat) = result
+
+
+# Make julia symbols (\bf* etc.) valid latex
+function texify(s)
+    return if occursin(r"^\\bf[A-Z]$", s)
+        replace(s, "\\bf" => "\\bm{\\mathrm{") * "}}"
+    elseif startswith(s, "\\bfrak")
+        replace(s, "\\bfrak" => "\\bm{\\mathfrak{") * "}}"
+    elseif startswith(s, "\\bf")
+        replace(s, "\\bf" => "\\bm{\\") * "}"
+    elseif startswith(s, "\\frak")
+        replace(s, "\\frak" => "\\mathfrak{") * "}"
+    else
+        s
+    end
+end
```

Comment:
This is very Tex-specific (like `uc2tex`), and so should be moved into `rendering/texformats.jl`, no ?

Reply:
wasn't sure whether this would be used anywhere else as well since many formats use latex symbols but I moved it. After all one would still likely go looking for something like this is the texformat file